src/data/doc-extensions/extensions.json

```diff
@@ -14,6 +14,20 @@
         "Browser"
       ]
     },
+    {
+      "name": "xk6-websockets",
+      "description": "WebSockets API specification implementation for k6",
```

Comment:
I would be more explicit here because we already have a WebSocket implementation in k6.

> New WebSocket API implementation that will replace the built `k6/ws` in the future.